spec/syntax.md

```diff
@@ -289,6 +289,8 @@ be preserved during formatting.
 ```abnf
 simple-start-char = content-char / s / "@" / "|"
 text-char         = content-char / s / "." / "@" / "|"
+quoted-char       = content-char / s / "." / "@" / "{" / "}"
+reserved-char     = content-char / "."
```

Comment:
These are already listed elsewhere in `syntax.md`. Is there a reason to bring them here, rather than keeping them in their current places?

Reply:
I'm not seeing this? Where are the productions doubled in @alerque's [text](https://github.com/alerque/message-format-wg/blob/update-abnf-snippets-in-spec/spec/syntax.md)?

Reply:
The `quoted-char` is currently under Literals right after `quoted`, and `reserved-char` under Reserved Annotations right after `reserved-body`. The proposed change moves their listings from those places (which are their only users) to here.

As is, their meaning is obvious from context. If they're moved here, then we should add some text here explaining what they are.

Reply:
Those definitions moved in the abnf source too, and I found their new/current locations much easier to grok than the old ones the syntax explanations were using because they are now grouped with more related context with other similar definitions.

If consensuses is that isn't a good change they can go back to their original places, but the abnf snippets will no longer be all blocks where the whole snippet comes from a contiguous bit of source. Having all the snippets match some contiguous bit from the original file makes it quite a bit easier to verify that it is in sync.

Reply:
As we do not intend to make further syntax changes, mirroring such changes in `syntax.md` as well as the ANBF should not need to be done later.

I have a weak preference for keeping these as they currently are, but I'm fine to go with whatever most people think.